Added 2 new titles to Books

stjepanjurekovic opened the pull request:
Hi,

Stjepan from Maning here. I thought these 2 latest Spring titles from our catalog might be a good match for your list. Thank you for considering it.

Best,

stjepanjurekovic committed:
Added 2 new titles to Books

Hi,

Stjepan from Maning here. I thought these 2 latest Spring titles from our catalog might be a good match for your list. Thank you for considering it.

Best,

ThomasVitale commented:
@stjepanjurekovic thanks for your contribution! Could you update the PR to include the titles so that they are alphabetically sorted? Also, since the previous edition of Spring in Action was already on the list, can you replace the existing one with the new one instead of adding a second item? Thank you!

stjepanjurekovic committed:
Added Spring Microservices in Action, 2nd Ed

Hi,

Stjepan from Manning here. I thought this title might be a good match for your list. Thank you for considering it.

Best,

ThomasVitale commented:
Thanks for your contribution. The books are now included in the list: https://github.com/ThomasVitale/awesome-spring/blob/main/README.md.
